### PR TITLE
Expose gcs to amara

### PR DESCRIPTION
**Pull request by markx**

### CODE REVIEW

File: service/storage.go [hunk 1]
```diff
@@ -43,13 +43,17 @@ func (gs *GCSStorage) Store(data []byte, filename string) (string, error) {
...
+		gs.logger.WithError(err).Error("error closing writer ")
+		return "", err
+	}
+	if err := obj.ACL().Set(ctx, storage.AllUsers, storage.RoleReader); err != nil {
+		gs.logger.WithError(err).Error("could not set ACL")
 		return "", err
 	}
 
-	return fmt.Sprintf("gs://%s/%s", gs.bucketName, objectFullName), nil
+	return fmt.Sprintf("https://storage.cloud.google.com/%s/%s", gs.bucketName, objectFullName), nil
```
Comment (fsouza) [inline service/storage.go]: Doesn't that require the bucket to be public?

We could generate a signed URL with a given expiration (making the expiration configurable): https://godoc.org/cloud.google.com/go/storage#hdr-Signed_URLs

Comment (markx) [inline service/storage.go]: I don't think it does. It only makes the object public.

Comment (fsouza) [inline service/storage.go]: Oh, that's cool. I don't know if making an unfinished captions file public is a good idea though, but we can go with that.

File: service/storage.go [hunk 2]
```diff
@@ -43,13 +43,17 @@ func (gs *GCSStorage) Store(data []byte, filename string) (string, error) {
...
-		gs.logger.WithError(err).Info("there was some error writing file")
+		gs.logger.WithError(err).Error("there was some error writing file")
 		return "", err
 	}
 	if err := writer.Close(); err != nil {
-		gs.logger.WithError(err).Info("error closing writer ")
+		gs.logger.WithError(err).Error("error closing writer ")
+		return "", err
+	}
+	if err := obj.ACL().Set(ctx, storage.AllUsers, storage.RoleReader); err != nil {
```
Comment (fsouza) [inline service/storage.go]: You can define the ACL in the writer and make one less operation in the GCS API :D 

```
writer.ObjectAttrs.ACL = []storage.ACLRule{{Entity: storage.AllUsers, Role: storage.RoleReader}}
```

Comment (markx) [inline service/storage.go]: 👍

**Pull request merged**